packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -182,12 +199,32 @@ TestProviders.propTypes = {
  * @param {object} children
  * @param {object} options
  */
-export const renderWithProviders = (children, options) =>
-    render(children, {
+export const renderWithProviders = (children, options) => {
+    const TestProvidersWithDataAPI = withReactQuery(withLegacyGetProps(TestProviders), {
```

Comment:
The naming of this Provider is a bit confusing. it does not reflect what it really is. Maybe TestProvidersWithReactQuery?

Reply:
I thought about that name too.. but we are using the data api "getProps" as well.. hence that name.

Reply:
Do we need to keep `withLegacyGetProps` at the end of the integration? If not, I think the name makes sense

Reply:
I think it's wise to leave the `withLegacyGetProps` in there as to catch any kind of integration errors if some come up. 

Lets leave the name as is for now.